include/glow/Backends/TraceEvents.h

```diff
@@ -65,7 +65,11 @@ struct TraceEvent {
                   const std::string &processName = "",
                   const std::map<int, std::string> &threadNames = {});
 
+  // Return the current time in microseconds in the timestamp domain.
   static uint64_t now();
+
+  // Return the thread id of the current thread.
```

Comment:
Is this correct? I think that this is more like a running counter that assigns a unique id for each thread.

Reply:
Well, what is that but a thread id? It's not the OS thread idea though to be sure (std::this_thread::id() isn't easily convertable to something human readable).

 I'll adjust the comment.